config/config.go

```diff
@@ -270,6 +274,52 @@ func (c *Config) Set(key string, value interface{}) {
 	c.onConfigChange()
 }
 
+func getAtomicMapKeys[T configTypes](v T, keys ...string) (string, string) {
+	sort.Strings(keys)
+	k := fmt.Sprintf("%T:%s", v, strings.Join(keys, ","))
+	return k, fmt.Sprintf("%s:%v", k, v)
+}
+
+func getOrCreatePointer[T configTypes](
+	mapPtr *map[string]any, dvsMapPtr *map[string]string, // using pointers to maps so that we can run "make" here
```

Comment:
Note to self to initialize the maps in the `Config` constructor to remove these pointers.